deploy/helm-chart/README.md

```diff
@@ -0,0 +1,81 @@
+# NGINX Kubernetes Gateway Helm Chart
+
+## Introduction
+
+This chart deploys the NGINX Kubernetes Gateway in your Kubernetes cluster.
+
+## Prerequisites
+
+- [Helm 3.0+](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+> Note: NGINX Kubernetes Gateway can only run in the `nginx-gateway` namespace. This limitation will be addressed in
+the future releases.
+> Note: NGINX Kubernetes Gateway does not yet have a published chart.
+
+1. Clone the repo and change into the `nginx-kubernetes-gateway` directory:
+
+   ```
+   git clone https://github.com/nginxinc/nginx-kubernetes-gateway.git
+   cd nginx-kubernetes-gateway
+   ```
+
+1. Install the Gateway API resources from the standard channel (the CRDs and the validating webhook):
+
+   ```
+   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.1/standard-install.yaml
+   ```
+
+1. Create the nginx-gateway Namespace:
+
+    ```
+    kubectl create ns nginx-gateway
+    ```
+
+### Installing the Chart from local
+
+To install the chart with the release name my-release (my-release is the name that you choose) into the nginx-gateway
+namespace:
+
+```
+helm install my-release ./deploy/helm-chart -n nginx-gateway
+```
+
+### Uninstalling the Chart
+
+To uninstall/delete the release `my-release`:
+
+```
+helm uninstall my-release -n nginx-gateway
+```
+
+The command removes all the Kubernetes components associated with the release and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the NGINX Kubernetes Gateway chart and their default values.
+
+|Parameter | Description | Default |
+| --- | --- | --- |
+|`gateway.nginxGateway.image.repository` | The repository for the NGINX Kubernetes Gateway image. | ghcr.io/nginxinc/nginx-kubernetes-gateway |
+|`gateway.nginxGateway.image.tag` | The tag for the NGINX Kubernetes Gateway image. | edge |
```

Comment:
This should probably just be the latest release version, right?

Reply:
I'm not sure, should we leave it at edge here and add the correct tags on the release commit, like we do for manifests? That way Helm is always deployable from edge as well (the plan is to have edge charts)?

Reply:
NIC provides edge helm charts, right? I guess as long as there is clear separation of helm repositories so edge and stable tags are easy to discern from a user perspective, it's probably fine.

Reply:
Yes, they do, but they aren't always useable because the tag is pinned to the previous release - meaning the templates may not align to the image used. Stable charts will be tagged according to what's in the Chart.yaml e.g. `0.1.0`, and the edge chart will be tagged `0.0.0-edge`. It will be the same repository - ghcr. The AC doesn't specify pushing to our own nginxinc managed helm repo - we can add it later though if it's required.